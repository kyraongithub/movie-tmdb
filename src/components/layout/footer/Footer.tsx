import React from 'react';

const Footer: React.FC = () => {
  return (
    <footer className="flex items-center justify-center p-4">
      &copy;code.kyra {new Date().getFullYear()}
    </footer>
  );
};

export default Footer;
