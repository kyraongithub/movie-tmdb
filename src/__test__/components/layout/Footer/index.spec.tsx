import { render } from '@testing-library/react';
import Footer from '../../../../components/layout/footer';

describe('Footer', () => {
  it('render Footer component', () => {
    const { container } = render(<Footer />);
    expect(container).toMatchSnapshot();
  });
});
