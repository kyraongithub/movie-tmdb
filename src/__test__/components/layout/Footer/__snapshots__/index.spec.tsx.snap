// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Footer > render Footer component 1`] = `
<div>
  <footer
    class="flex items-center justify-center p-4"
  >
    Â©code.kyra 
    2025
  </footer>
</div>
`;
